webpack.config.shared.js

```diff
@@ -57,7 +57,11 @@ module.exports = {
     exclude: /node_modules/,
     loader: 'babel-loader',
     query: {
-      "presets": ["latest", "react"],
+      "presets": [
+        ["env", { "modules": false }],
```

Comment:
Yes, it's turned on by this PR (it wasn't before, it is very poorly documented which is quite frustrating). It won't have any effect because (as per the issue above) it is broken currently.